docs/how-to/dss/install-dss-cli.rst

```diff
@@ -0,0 +1,29 @@
+Install DSS
+===========
+
+This guide explains how to install the Data Science Stack (DSS) Command Line Interface (CLI).
+
+Overview
+--------
+
+The DSS CLI is distributed as a snap accessible from the [snap store](https://snapcraft.io/data-science-stack).
+
```

Comment:
I am missing prerequisites section where we mention that you need microk8s. Plus we might want to mention that you need ubuntu for snaps.